secedgar/cli.py

```diff
@@ -8,8 +8,14 @@
 
 
 @click.group()
-def cli():
+@click.option('--user-agent',
+              help='Value used for HTTP header "User-Agent" for all requests.',
+              required=True)
+@click.pass_context
```

Comment:
The `--user-agent` option will be made available to the commands via "context passing", as described in this documentation: https://click.palletsprojects.com/en/7.x/commands/#nested-handling-and-contexts